packages/py-moose-lib/moose_lib/dmv2/materialized_view.py

```
@@ -5,14 +5,23 @@
 
 from ..blocks import ClickHouseEngines
-from ..utilities.sql import quote_identifier
 from .types import BaseTypedResource, T
 from .olap_table import OlapTable, OlapConfig
-from .sql_resource import SqlResource
+from ._registry import _materialized_views
+from ._source_capture import get_source_file_from_stack
+from .view import View
+
+
+def _format_table_reference(table: Union[OlapTable, View]) -> str:
```

Ruff (0.14.8)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)